### PR TITLE
Document: CC to AKO migration

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

File: docs/cc_to_ako.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Cloud connector to AKO migration
+## Overview
+Cloud connector(CC) is not supported from Avi Controller versrion 18.2.10 onwards. To migrate existing workload present in kubernetes cluster from cloud connector to AKO, following two features will be used.
```
Comment (chauhanshubham) [inline docs/cc_to_ako.md:3]: typo: versrion -> version

File: docs/cc_to_ako.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+* <b>Namespace Sync feature</b>
+
+    Namespace sync feature allows K8 objects from specific namespace to be synced with Avi-Controller. For that, namespace has to be labelled with same key:value pair as that labelKey and labelValue mentioned in values.yaml. This feature is supported by AKO from 1.4.1. 
+    
+    Details about this feature is at: [Namespace Sync in AKO](objects.md#namespace-sync-in-ako)
+
+So crux of migration activity to remember is to use same "key:value" pair 
+
+1. as an exclusion attributes in Cloud connector setup.
```
Comment (chauhanshubham) [inline docs/cc_to_ako.md:19]: Nit: attribute (singular)

File: docs/cc_to_ako.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+*  Setup a new Vcenter cloud with IPAM.
+*  Setup a DNS Service
+*  Replicate AVI side objects referred by Ingresses/Services as part of AVI_PROXY annotations to new           controller.
+
+### 2. Deploy AKO CRDs
+
+For each AVI_PROXY annotation present in an ingress, create corresponding AKO Http rule/Host rule/ AviInfrasetting rule. 
+Details of these CRDs can be found out [here](crds/overview.md)
+
+If any AVI_PROXY annotation is not supported by these CRD's, then VS which is migrated to AKO will not have same features as that in CloudConnector.
```
Comment (chauhanshubham) [inline docs/cc_to_ako.md:44]: Nit: CRDs

File: docs/cc_to_ako.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+### 2. Deploy AKO CRDs
+
+For each AVI_PROXY annotation present in an ingress, create corresponding AKO Http rule/Host rule/ AviInfrasetting rule. 
+Details of these CRDs can be found out [here](crds/overview.md)
+
+If any AVI_PROXY annotation is not supported by these CRD's, then VS which is migrated to AKO will not have same features as that in CloudConnector. 
+
+### 3. AKO Deployment
+
+Deploy AKO (version 1.4 and above) with namespace sync feature enabled.
```
Comment (chauhanshubham) [inline docs/cc_to_ako.md:48]: I think it'll be good to have a link that describes how to set t he namespace sync feature. `values.md` probably.

**Review by chauhanshubham — COMMENTED** [review]
couple of nits.

File: docs/cc_to_ako.md (hunk 4 above)
Comment (akshayhavile) [reply docs/cc_to_ako.md:48]: Screen shot pasted for this section shows how to set namespace sync. Also we have given link of page that explains how to perform namespace sync.

**Pull request closed without merging**